blackbox/docs/sql/reference/constraints.txt

```diff
@@ -52,3 +52,24 @@ column constraint or a table constraint.
 
 For further details about the meaning of the options see
 :ref:`indices_and_fulltext`.
+
+
+.. _column_constraints:
+
+==================
+COLUMN CONSTRAINTS
+==================
+
+Column constraints are constraints that are applied on each column of the table separately.
+The supported column constraints are:
+
+ * :ref:`not_null_constraint`
```

Comment:
would probably make sense to refer to the primary key constraint here too, since there is a table and a column version